docs/widgetbook-cloud/reviews.mdx

```diff
@@ -1,124 +1,58 @@
-1. Add new or change existing Widgetbook use cases in your feature branch. Note: Without changed use cases, the review feature won't work, as there would be nothing to review.
-1. Optionally, [add addons and knobs](/widgetbook-cloud/reviews#how-to-set-up-addons-and-knobs) as well as [Figma design links](/widgetbook-cloud/reviews#how-to-add-a-figma-design-link).
-1. Publish a review for the feature branch with changed Widgetbook use cases using this command: `widgetbook publish --api-key <API> --base-branch main`. This will publish both a **build and a review** for the latest commit in your feature branch.
-
-<Info>
-    Most CI/CD providers expose the `BASE-BRANCH` via environment variables for pull-requests. Check our [CI/CD Setup guide](/widgetbook-cloud/setup) for more insights.
-</Info>
-
-### Example
-
-```bash
-# Install dependencies
-flutter pub get
-# !!! 
-# !!! Run the build runner
-# !!! This is mandatory even if you have committed the generated files since 
-# !!! the generator creates files that are consumed by the Widgetbook CLI
-# !!!
-flutter pub run build_runner build --delete-conflicting-outputs
-# Build the Widgetbook
-flutter build web -t lib/ui_catalogs.widgetbook.dart
-# Install the CLI
-dart pub global activate widgetbook_cli
-# Run the CLI
-widgetbook publish --api-key API-KEY --base-branch main
-```
-
-## Review Information
-To populate a review, information regarding the changes in use cases is mandatory. When the `--base-branch` argument is used, the CLI fetches details about the altered use cases.
-
-<Info>
-Widgetbook Cloud showcases Reviews for all pull requests. But, only those Reviews with use case alterations are accessible.
-</Info>
 <Warning>
-At present, Widgetbook Cloud exclusively displays use cases that underwent changes, additions, or removals within the feature branch compared to the base branch. Consequently, existing use cases, even if unreviewed, won't be displayed.
+  Widgetbook Cloud Review currently only works with GitHub, we are planning to
+  add more Git providers in the future.
 </Warning>
 
+Widgetbook Cloud introduces a _Review_ feature, a zero-configuration UI regression detection tool that you can get out-of-the-box if you have Widgetbook setup for your project.
+
+| GitHub PRs                                                       | Widgetbook Cloud Reviews                                             |
+| ---------------------------------------------------------------- | -------------------------------------------------------------------- |
+| Compare **code changes** between `base` commit and `head` commit | Compare **visual differences** between `base` build and `head` build |
+
+## Pre-requisites
+
+1. Connect your Widgetbook Project to a GitHub repo from the project settings page.
+1. Make sure to you have a [Widgetbook Build](/widgetbook-cloud/hosting) on pushes of new commits via CI (e.g. GitHub Actions).
+1. _[Optional]_ [Add Figma URL](#widgetbook-x-figma) to your use-cases.
```

Comment:
This link leads to the overview page. Here, the user can't find information on how to add Figma